### PR TITLE
Fix warm restart test left neighbors on vEthernet(s)

### PR DESCRIPTION
**Pull request by qiluo-msft**
Fixed https://github.com/Azure/sonic-swss/issues/596

### CODE REVIEW

**Comment by lguohan** [comment]
why does warm reboot test left such arp entries in the first place? do you know?

File: tests/test_warm_reboot.py [hunk 1]
```diff
@@ -146,6 +146,11 @@ def test_PortSyncdWarmRestart(dvs):
     (status, fvs) = neighTbl.get("Ethernet20:11.0.0.10")
     assert status == True
 
+    # Make sure no neighbors on vEthernet
+    keys = neighTbl.getKeys();
+    for key in keys:
+        assert not key.startswith("vEthernet")
```
Comment (lguohan) [inline tests/test_warm_reboot.py]: this can be added to a appdb_validator in conftest.py, it is generic.

**Comment by qiluo-msft** [comment]
The root cause is not clear to me.

File: tests/test_warm_reboot.py (hunk 1 above)
Comment (qiluo-msft) [reply tests/test_warm_reboot.py]: Resolved

**Comment by qiluo-msft** [comment]
Retest this please

**Comment by lguohan** [comment]
retest this please

File: tests/conftest.py [hunk 2]
```diff
@@ -73,20 +89,24 @@ def __init__(self, ctn_name, pid, i):
...
+            ensure_system("ip netns exec %s ip link set dev eth0 up" % (self.nsname))
+            ensure_system("ip netns exec %s ethtool -K eth0 tx off" % (self.nsname))
 
             # bring up link in the virtual switch
-            os.system("nsenter -t %d -n ip link set dev %s up" % (pid, self.vifname))
+            ensure_system("nsenter -t %d -n ip link set dev %s up" % (pid, self.vifname))
+
+        # disable arp, so no neigh on vEthernet(s)
+        # Note: outside the if-else, so existing VS container could be fixed
+        ensure_system("nsenter -t %d -n ip link set arp off dev %s" % (pid, self.vifname))
```
Comment (jipanyang) [inline tests/conftest.py]: I think arp is needed for neighborsyncd and fdb function test?  The vEthernet interfaces here are actually mapped to Ethernet interfaces in vs (thought I'm not clear how that is done with the docker --network container:sw magic)

**Comment by jipanyang** [comment]
Maybe the problem is because in the test, vs and sw dockers instances are re-used.  Those vEthernet interfaces are created at the very beginning using create_vnet.sh script.  The stale arp entries could be cleared before starting new round of test run.

File: tests/conftest.py (hunk 2 above)
Comment (qiluo-msft) [reply tests/conftest.py]: I disabled arp on only vEthernetXX. ARP and neighbors on EthernetXX are not affected during my experiment.

**Pull request merged**